src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -183,16 +185,32 @@ def test_create_osds(self, _send_command, _get_connection, _save_host, _rm_host,
             )
         ])
     ))
+    @mock.patch("cephadm.module.CephadmOrchestrator.send_command")
+    @mock.patch("cephadm.module.CephadmOrchestrator.mon_command", mon_command)
+    @mock.patch("cephadm.module.CephadmOrchestrator._get_connection")
     @mock.patch("cephadm.module.HostCache.save_host")
     @mock.patch("cephadm.module.HostCache.rm_host")
-    def test_remove_osds(self, _save_host, _rm_host, cephadm_module):
+    @mock.patch("cephadm.osd.RemoveUtil.get_pg_count", lambda _, __: 0)
+    def test_remove_osds(self, _send_command, _get_connection, _save_host, _rm_host, cephadm_module):
```

Comment:
Is the patching order correct? For most parts of the files.
The decorators are applied from bottom up.
But it seems it doesn't matter because the parameters are not used in the calls.

Reply:
I also mentioned that when writing unittests.. As you mentioned though, they aren't used.

Reply:
see https://github.com/ceph/ceph/pull/33663